src/shell/shell_misc.cpp

```diff
@@ -331,7 +348,9 @@ void DOS_Shell::InputCommand(char * line) {
 				if (l_completion.size() && it_completion->length()) {
 					for (;str_index > completion_index; str_index--) {
 						// removes all characters
-						outc(8); outc(' '); outc(8);
+						backone();
+						outc(' ');
+						backone();
 					}
```

Comment:
Thanks for all nice these formatting improvements!
It's now much easier and more obvious that three operations are happening here, when quickly skimming over.